crates/turborepo-ui/src/tui/term_output.rs

```diff
@@ -7,7 +7,15 @@ use super::{
     Error,
 };
 
-const SCROLLBACK_LEN: usize = 1024;
+const SCROLLBACK_LEN: usize = 2048;
```

Comment:
Making sure to call extra attention here: Users mentioned they felt 1024 was too small, so I doubled it here. Not sure about if there's a reason I shouldn't do this or if a different value should be used. Just did this for the sake of discussion.

Reply:
I think this is okay since https://github.com/vercel/turborepo/pull/9123 has been merged.

`vt100` used to need to iterate through every scrollback row in order to render a visible row so 1024->2048 would be a noticeable perf hit. I will do a quick profile to check if there's a noticeable impact from this change.